### ISSUE TITLE
v4 Page view update inventory

### ISSUE THREAD

**Issue by CGillen** [osulp/Scholars-Archive#2742]
### Descriptive summary

This will be an epic ticket taking inventory of pages to update. We can re-use this checklist when we move to v5 to double check we caught all the pages that need css/bootstrap/views updates

### Accessibility Concerns

All of it. Each PR should definitely have a review

**Comment by straleyb** [comment]
-----------------------DASHBOARD----------------------------------------------------------------
- [x] Header text is too big
- [x] Login link needs to be top aligned
- [x] Scholars Archive Home link is too big
- [x] Dashboard Sidebar choices need to be spaced out
- [ ] Repo growth graph needs to be stretched
- [ ] Graphs below repo growth need to be the same length
- [ ] Admin Sets Graph title needs to be fixed and bolded
- [x] Filter Works button in Works Dashboard colored incorrectly
- [x] Collections dashboard filter button colored incorrectly
- [ ] Something went wrong page not rendering CSS correct (not compiled?)
- [ ] Profile Dashboard numbers not in grey ovals and not placed correctly on page
- [ ] Notifications Table Title (top section) search bar not placed correctly
- [ ] Transfers titles not bold and too small
- [ ] Review Submissions table header layout incorrect. Search bar and some text.
----------------------------HOMEPAGE-----------------------------------------------------------
- [x] Homepage Missing Image in header
- [x] Homepage Search bar misplaced
- [x] Homepage Hover over links not turning grey
- [x] All dropdown on homepage not colored correctly

**Comment by straleyb** [comment]
Ill keep continue adding to this list

**Comment by straleyb** [comment]
----------------------------SEARCH RESULTS PAGE-----------------------------------------------------------
- [x] Facets Need to be adjacent with no space in between them
- [x] Facet card titles need to be more bold
- [x] Limit your search button color for chosen layout incorrect color
----------------------------WORK SHOW PAGE-------------------------------------------------------------
- [x] Badges Too small

**Comment by simholt** [comment]
- [x] Featured works panels should all be the same width.

<img width="683" height="495" alt="Image" src="https://github.com/user-attachments/assets/f504935b-4218-48d8-85d8-1676b86d1b70" />

**Comment by simholt** [comment]
- [x] Visibility box on edit page --> sherpa/romeo has changed, so update link and text:

Change to: 
Jisc
https://openpolicyfinder.jisc.ac.uk/

<img width="272" height="345" alt="Image" src="https://github.com/user-attachments/assets/120e0af4-7028-4104-b701-44e80dfd0eba" />
